netstandard/pkg/NETStandard.Library.pkgproj

```diff
@@ -31,6 +33,10 @@
     <File Include="..\tools\targets\netstandard1.7\$(TargetsFileName)">
       <TargetPath>build/netstandard1.7/$(Id).targets</TargetPath>
     </File>
+
+    <Dependency Include="_._">
```

Comment:
No, other netstandard versions get the dependencies specified by the imported props file.  This is a placeholder dependency to create an empty dependency group starting with `netstandard1.7`.  Once we move everything to `netstandard2.0` this will change to that TFM.